### DISCUSSION TITLE
How do I revoke access to individual users ahead of the removal date?

### DISCUSSION THREAD

**Discussion by cm-howard** [community/community#51989] [Copilot Conversations]
### Select Topic Area

Question

### Body

I've recently been providing access to specific teams within an Org for Copilot use. 

When revoking access I'm informed for each team that this will be revoked on Day 30 (end of the current billing period). However, I want to ensure that access is revoked for that team ahead of this date and not wait for the automatic removal for compliance reasons.

In order to attempt to do this, I was therefore removing individuals from the Team that that had access provided, however these users now show as individuals within the Copilot list of approved users outside of the Team they were once in and have the original revoke date alongside them.

In summary, how can I guarantee that a users access to Copilot is removed ahead of the final day of the billing cycle?
